### PR TITLE
Add more API Kind and SKU names

### PR DESCRIPTION
**Pull request by felixwa**
Add more API Kind and SKU names.
Model API Kind and SKU as string instead of enum. This would give us and users more flexibility.
Add related unit tests.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@felixwa__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (felixwa). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

File: src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/Generated/Models/CognitiveServicesAccountCreateParameters.cs [hunk 1]
```diff
@@ -47,11 +47,14 @@ public CognitiveServicesAccountCreateParameters(Sku sku, Kind kind, string locat
...
-        /// 'LUIS', 'Recommendations', 'Speech', 'TextAnalytics', 'WebLM'
+        /// Possible values include: 'Academic', 'Bing.Autosuggest',
+        /// 'Bing.Search', 'Bing.Speech', 'Bing.SpellCheck',
+        /// 'ComputerVision', 'ContentModerator', 'Emotion', 'Face', 'LUIS',
+        /// 'Recommendations', 'SpeakerRecognition', 'Speech',
+        /// 'SpeechTranslation', 'TextAnalytics', 'TextTranslation', 'WebLM'
         /// </summary>
         [JsonProperty(PropertyName = "kind")]
-        public Kind Kind { get; set; }
+        public string Kind { get; set; }
```
Comment (cormacpayne) [inline src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/Generated/Models/CognitiveServicesAccountCreateParameters.cs]: @felixwa changing the type of this property is a breaking change

File: src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/Generated/Models/Sku.cs [hunk 2]
```diff
@@ -29,19 +29,19 @@ public partial class Sku
...
         /// <summary>
         /// Gets or sets the sku name. Required for account creation, optional
-        /// for update. Possible values include: 'F0', 'S0', 'S1', 'S2',
-        /// 'S3', 'S4'
+        /// for update. Possible values include: 'F0', 'P0', 'P1', 'P2',
+        /// 'S0', 'S1', 'S2', 'S3', 'S4', 'S5', 'S6'
         /// </summary>
         [JsonProperty(PropertyName = "name")]
-        public SkuName Name { get; set; }
+        public string Name { get; set; }
```
Comment (cormacpayne) [inline src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/Generated/Models/Sku.cs]: @felixwa changing the type of this property is a breaking change

**Review by cormacpayne — CHANGES_REQUESTED** [review]
@felixwa Hey Felix, I left a couple of comments. Since CognitiveServices is in preview right now, it is OK to make these breaking changes.

A few more comments:

1) Was this code generated from a swagger spec? If so, you will need to specify which version of AutoRest you used, as well as the swagger spec used, in the `generate.cmd` file located [here](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/generate.cmd)

2) You will need to make a version bump because of the changes you are making. Since there are breaking changes, it looks like you would go to 0.2.0-preview. You will need to make the changes in [project.json](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/project.json) and [AssemblyInfo.cs](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/Properties/AssemblyInfo.cs)

**Comment by felixwa** [comment]
@cormacpayne , thanks for the comments.

1. I have changed generate.cmd file to reflect the AutoRest version used to generate this SDK. 
2. The Swagger file URL is still that one though, we made similar changes to that file and it is still under review. 
3. Also I changed the version to 2.0 as you suggested.

Please review.

File: src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/Generated/Models/Sku.cs (hunk 2 above)
Comment (felixwa) [inline src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/Generated/Models/Sku.cs]: Change the version number as you suggested.

File: src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/Generated/Models/CognitiveServicesAccountCreateParameters.cs (hunk 1 above)
Comment (felixwa) [inline src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/Generated/Models/CognitiveServicesAccountCreateParameters.cs]: Change the version number as you suggested.

**Comment by felixwa** [comment]
@cormacpayne, thanks for the comments, I have changed version of AutoRest in generate.cmd and Assembly version to 2.0 in project.json/AssemblyInfo.cs.

Please review and let me know if you have any other concerns.

-Felix

**Comment by cormacpayne** [comment]
@felixwa Hey Felix, since you used version 0.17.x of AutoRest from a few months ago, would you mind using one of the latest nightly builds to regenerate your code? We are up to version 1.0.0, and you can find out how to access it [here](https://github.com/Azure/autorest#getting-autorest)

**Comment by felixwa** [comment]
@cormacpayne , now I have used AutoRest 1.0.0 20170101 to regenerate all model files.

**Comment by felixwa** [comment]
@cormacpayne, ping...API Spec side had accepted and merged my change, can you review and take a look at this one at the earliest? So that I can continue to modify PS for Cognitive Services.

**Review by cormacpayne — APPROVED** [review]

**Pull request merged**